### ISSUE TITLE
NPUA80643 Ratchet & Clank failing to load from a savestate

### ISSUE THREAD

**Issue by redlava11** [RPCS3/rpcs3#13322]
### Quick summary

Get an error when trying to load from a savestate. Done it a few times over the course of a few days, worked without any issues. But now every time I try to load the savestate I get this.
F {RSX.W4} SIG: Thread terminated due to fatal error: Verification failed
(in file C:\Users\ContainerAdministrator\AppData\Local\Temp\cirrus-ci-build\rpcs3\Emu\RSX\VK\VKPipelineCompiler.cpp:125[:3], in function int_compile_graphics_pipe)
![image](https://user-images.githubusercontent.com/56317167/215349299-7234e96e-ed40-4e97-a9e8-8a5132112fb1.png)
![image](https://user-images.githubusercontent.com/56317167/215349279-1f2667b9-86b1-47ce-bd7f-96d8ce52fc21.png)

Game works fine when booting normally.

### Attach a log file

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/10530573/RPCS3.log.gz)

### System configuration

OS: Windows 10
CPU: Ryzen 5 2600X
GPU: GTX 1080Ti
RAM: 16GB

### Other details

Would be great to get it booting from that same savestate since I have a lot of bolts in it and my dumb ass didn't save the game manually.

**Comment by Triticum0** [comment]
Use the older build you were on before an try to save the game.

**Comment by redlava11** [comment]
Older build fixed the issue. Thank you very much!

**Comment by AniLeo** [comment]
Of course using an old build isn't a fix.

**Comment by redlava11** [comment]
Yeah my bad, go too excited to have the issue resolved. Can still do some testing on the latest build if needed

**Comment by Darkhost1999** [comment]
This needs to be retested on the current master and up-to-date logs provided need information since 02/2023

**Comment by digant73** [comment]
@redlava11 is that still an issue?
